### ISSUE TITLE
After downloading template and reuploading TR-300 error appears

### ISSUE THREAD

**Issue by reijovosu** [bigcommerce/stencil-cli#632]
For BigCommers newcomers and also beginners it is unacceptable behaviour, when built in features are not working out of the box. 

### Expected behavior
Cornerstone template should be able to upload

### Actual behavior
Gets TR-300 error

### Steps to reproduce behavior
In store templates I have blank copy of Cornerstone template

Inside empty directory:
`$ stencil init --url https://[myurl].mybigcommerce.com --token [mytoken]`
I get an error:
```
fs.js:114
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/Users/reijo/Projects/oneills2/config.json'
    at Object.openSync (fs.js:443:3)
    at Object.readFileSync (fs.js:343:35)
    at ThemeConfig.getRawConfig (/usr/local/lib/node_modules/@bigcommerce/stencil-cli/lib/theme-config.js:418:30)
    at ThemeConfig.getConfig (/usr/local/lib/node_modules/@bigcommerce/stencil-cli/lib/theme-config.js:57:23)
    at /usr/local/lib/node_modules/@bigcommerce/stencil-cli/lib/stencil-init.js:35:60
    at /usr/local/lib/node_modules/@bigcommerce/stencil-cli/node_modules/graceful-fs/graceful-fs.js:61:14
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)

```
Thats OK. it is not actually braking anything.

`$ stencil download`

After editing one file

`$ stencil push`
And error:
```
Validating theme...
SCSS Parsing Started...
Template Parsing Started...
Language Files Parsing Started...
Building Theme Schema File...
Schema Translations Parsing Started...
ok -- Schema Translations Parsing Finished
Theme task Started...
ok -- Language Files Parsing Finished
ok -- Theme Schema Building Finished
ok -- SCSS Parsing Finished
ok -- Template Parsing Finished
Webpack Bundle Analyzer saved report to /Users/[myusername]/Projects/oneills2/assets/dist/report.html

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (300 KiB).
This can impact web performance.
Assets: 
  theme-bundle.main.js (303 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (300 KiB). This can impact web performance.
Entrypoints:
  main (303 KiB)
      theme-bundle.main.js

ok -- Theme task Finished
Generating Manifest Started...
ok -- Manifest Generation Finished
ok -- Zipping Files Finished
ok -- Theme Upload Finished
Processing [===========                                                                                    ] 12%; ETA: 0.0

not ok -- JobCompletionStatusCheckError: Job Failed see details below:
There was a problem with the zip file. (TR-300)

Please visit the troubleshooting page https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads
If this error persists, please visit https://github.com/bigcommerce/stencil-cli/issues and submit an issue.
```

**Comment by junedkazi** [comment]
Have you looked at this https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#tr-300-error-upon-theme-upload

**Comment by reijovosu** [comment]
Yes I have.
1) For normal user this error should not exist.
2) Where is this bundle.js.map and why I need to delete it?

Note: I am downloading built in Cornerstone template and editing only one file.

**Comment by MaxGenash** [comment]
@reijovosu
Sorry for the long response.
I believe your problems may be caused by an outdated version of stencil-cli and/or Cornerstone.
Can you recheck please using the latest versions of the libraries?

**Comment by reijovosu** [comment]
Nope. Both are up to date. But deleting or editing https://github.com/bigcommerce/cornerstone/tree/master/meta files will fix the problem.

So the error is correct.

**Comment by MaxGenash** [comment]
Ok, thanks for your report.
I created a corresponding ticket for this issue: STRF-8851.

**Comment by chandlertrinity** [comment]
I had the exact problem and deleting the meta files was the fix!

Thanks

**Comment by gevvvvv** [comment]
@chandlertrinity this fixed it for me as well.

**Comment by reijovosu** [comment]
Just out of curiosity. How hard it is to resize obviously too big preview images in cornerstone template?

**Comment by MaxGenash** [comment]
@reijovosu @chandlertrinity
This issue was fixed in https://github.com/bigcommerce/stencil-cli/pull/685 and released as v3.1.1 to NPM.
Please, retry with the updated version of stencil-cli.

Basically, there was a bug in unarchiving code inside `stencil download` command which was corrupting images and causing TR-300 on `stencil push`.

So if you previously downloaded the theme using `stencil download` command - you will need to delete/rollback it and download a normal not broken version

**Comment by reijovosu** [comment]
OK. Thank you
